saw-remote-api/src/SAWServer.hs

```diff
@@ -220,6 +220,7 @@ initialState readFileFn =
                 , rwDebugIntrinsics = True
                 , rwWhat4HashConsing = False
                 , rwWhat4HashConsingX86 = False
+                , rwWhat4Eval = False
```

Comment:
Let's open  ticket about plumbing this new option through the RPC server.

Reply:
For what it's worth, it's fairly easy to add support for toggling options like this in `saw-remote-api`. You basically just add a new constructor [here](https://github.com/GaloisInc/saw-script/blob/master/saw-remote-api/src/SAWServer/SetOption.hs#L44-L49), and then update the code in `setOption`, `parseOption`, and `instance Doc.DescribedMethod SetOptionParams OK` accordingly.